azurerm/resource_arm_virtual_machine_scale_set_test.go

```diff
@@ -174,6 +174,67 @@ func TestUnitAzureRMVirtualMachineScaleSet_basicPublicIP_simpleUpdate(t *testing
 	})
 }
 
+func TestUnitAzureRMVirtualMachineScaleSet_updateNetworkProfile(t *testing.T) {
```

Comment:
Hi @alexsapran can we update the test function name to start with `TestAccAzureRMVirtualMachineScaleSet` instead of `TestUnitAzureRMVirtualMachineScaleSet`